web/client/components/map/openlayers/__tests__/DrawSupport-test.jsx

```diff
@@ -22,6 +23,14 @@ describe('Test DrawSupport', () => {
         setTimeout(done);
     });
 
+    /**
+     * used to reduce a bit the boilerplate and ReactDom stuff
+     * TODO extend it to the other tests
+    */
+    const renderDrawSupport = (props = {}) => {
+        ReactDOM.render(<DrawSupport {...props}/>, document.getElementById("container"));
```

Comment:
You don't reduce the boilerplate if you use it only in one place / test.
It's useful to refactor code in external functions if they are used at least in a bunch of different places, expecially in tests, where being able to read the full test at once is more important than duplication.
If you have too much code in one test it's better to split it in two different tests.

Reply:
I created this function Because i wanted to write less code  in particular the parts
ReactDOM.render(<COMPONENT {props}/>, document.getElementById("container"));
I could generalize it so we can use it in general and also in other places, but I see that you prefer the older version.
Going to refactor this.